### PR TITLE
added ingest certifyPkg for both backends

### PR DESCRIPTION
**Pull request by pxp928**
- Added ingest `certifyPkg` for both in-memory and neo4j
- Will create another PR to update the existing "verbs" to ingest only at the `pkgVersion` level

### CODE REVIEW

File: pkg/assembler/graphql/schema/certifyBad.graphql [hunk 1]
```diff
@@ -54,6 +54,17 @@ PkgSrcArtObject is a union of Package, Source and Artifact. Any of these objects
...
+enum PkgMatchType {
+  ALL_VERSIONS
+  SPECIFIC_VERSION
+}
+
+input MatchFlags {
+  pkg: PkgMatchType!
+  depPkg: PkgMatchType!
+}
+
```
Comment (mihaimaruseac) [inline pkg/assembler/graphql/schema/certifyBad.graphql:67]: These seem unused now. Will this be in the next PR?

**Review by mihaimaruseac — APPROVED** [review]

File: pkg/assembler/graphql/schema/certifyBad.graphql (hunk 1 above)
Comment (pxp928) [reply pkg/assembler/graphql/schema/certifyBad.graphql:67]: I can remove it but yes I was going to work on `CertifyBad` next.

Comment (pxp928) [reply pkg/assembler/graphql/schema/certifyBad.graphql]: removed.

Comment (pxp928) [reply pkg/assembler/graphql/schema/certifyBad.graphql]: made a note of it for the next PR all good :)

File: pkg/assembler/backends/testing/certifyPkg.go [hunk 2]
```diff
@@ -82,28 +83,90 @@ func registerAllCertifyPkg(client *demoClient) error {
...
+			"IngestCertifyPkg :: multiple package found")
+	}
+
+	depPkgQualifiers := []*model.PackageQualifierSpec{}
+	for _, quali := range pkg.Qualifiers {
+		pkgQualifier := &model.PackageQualifierSpec{
+			Key:   quali.Key,
+			Value: &quali.Value,
+		}
+		pkgQualifiers = append(pkgQualifiers, pkgQualifier)
```
Comment (lumjjb) [inline pkg/assembler/backends/testing/certifyPkg.go:145]: is this supposed to be depPkgQualifiers?

**Review by lumjjb — APPROVED** [review]
1 comment, else lgtm

File: pkg/assembler/backends/testing/certifyPkg.go (hunk 2 above)
Comment (pxp928) [reply pkg/assembler/backends/testing/certifyPkg.go]: yes! Fixed.

**Pull request merged**